src/theme/components/notification.ts

```diff
@@ -121,6 +121,11 @@ const NotificationContent = {
             color: "dark.ui.link.secondary",
           },
         },
+        _focus: {
```

Comment:
This general pattern would be great to have a function where the color is passed as an argument. There is a function which sort of does this already that _might_ be usable here: https://github.com/NYPL/nypl-design-system/blob/development/src/theme/components/global.ts#L7

If it's not useable here, that's okay, but make sure that this focus object also includes a `_dark` option.